docs/api/rebac/groups.mdx

```diff
@@ -6,60 +6,86 @@ title: Groups API
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-The Groups API allows you to manage your resources and users within groups. You can create, update, delete, and list groups.
-Groups enable you to easily group together resources, resource-instances, and users. In addition groups simplify ReBAC permissions for users, by allowing roles to be assigned to the groups themselves.
-You can assign roles to a group and then add users to the group through the Groups API.
-The users within the group will derive roles form the roles assigned to the group .
+The Groups API simplifies permission management for your resources and users by using groups. Groups can include resources, resource instances, and users, enabling you to set permissions for everything contained within a group at once.
```

Comment:
"Groups can include resources, resource instances..." - you can group resource types to set up the scheme of your app resources, later the permission will be managed on resource instances groups according the the scheme that was set on the type. so when saying "Groups can include resources, resource instances and users..." I think it communicates the wrong idea. 

" enabling you to set permissions for everything contained within a group at once" - only relevant for user groups. I think it's confusing in the context of resource groups.

Reply:
For the first note: I agree. Changed to: "These can be used to group together resources, resource instances, or users,"

As for the second note, how's: "enabling you to set a unified permission scheme for everything contained within a group"? I need something that works for all three group types..